crates/cli/src/fs.rs

```diff
@@ -1,13 +1,21 @@
-use std::{io, os, path::Path};
+use std::{io, path::Path};
 
 #[cfg(unix)]
 pub fn symlink_dir(original: &Path, link: &Path) -> io::Result<()> {
-    os::unix::fs::symlink(original, link)
+    std::os::unix::fs::symlink(original, link)
 }
 
 #[cfg(windows)]
 pub fn symlink_dir(original: &Path, link: &Path) -> io::Result<()> {
-    os::windows::fs::symlink_dir(original, link)
+    match std::os::windows::fs::symlink_dir(original, link) {
+        Ok(_) => Ok(()),
+        Err(_) => {
```

Comment:
Would it makes sense to look into the error type in order to create a junction for performance reasons?

Reply:
Something like creating a junction only when we have the elevated privileges error?

Reply:
Yes

Reply:
I added another commit, now it searchs for the error code `1314`.